src/muya/lib/ui/quickInsert/config.js

```diff
@@ -47,108 +66,125 @@ export const quicInsertObj = {
   //   color: 'rgb(224, 54, 54)'
   // }],
   'basic block': [{
-    title: 'Text',
-    subTitle: 'Just start write plain text.',
+    title: 'Paragraph',
+    subTitle: 'Lorem Ipsum is simply dummy text',
     label: 'paragraph',
+    shortCut: `${COMMAND_KEY}0`,
     icon: paragraphIcon,
     color: 'rgb(224, 54, 54)'
   }, {
     title: 'Horizontal Line',
-    subTitle: 'A horizontal line inserted.',
+    subTitle: '---',
     label: 'hr',
+    shortCut: `⌥${COMMAND_KEY}-`,
     icon: hrIcon,
     color: 'rgb(255, 83, 77)'
   }, {
     title: 'Front Matter',
-    subTitle: 'Just like Front Matter in hexo.io.',
+    subTitle: '--- Lorem Ipsum ---',
     label: 'front-matter',
+    shortCut: `⌥${COMMAND_KEY}Y`,
     icon: frontMatterIcon,
     color: 'rgb(37, 198, 252)'
   }],
   'header': [{
     title: 'Header 1',
-    subTitle: 'Header rendered by h1.',
+    subTitle: '# Lorem Ipsum is simply ...',
     label: 'heading 1',
+    shortCut: `${COMMAND_KEY}1`,
     icon: header1Icon,
     color: 'rgb(86, 163, 108)'
   }, {
     title: 'Header 2',
-    subTitle: 'Header rendered by h2.',
+    subTitle: '## Lorem Ipsum is simply ...',
     label: 'heading 2',
+    shortCut: `${COMMAND_KEY}2`,
     icon: header2Icon,
     color: 'rgb(94, 133, 121)'
   }, {
     title: 'Header 3',
-    subTitle: 'Header rendered by h3.',
+    subTitle: '### Lorem Ipsum is simply ...',
     label: 'heading 3',
+    shortCut: `${COMMAND_KEY}3`,
     icon: header3Icon,
     color: 'rgb(119, 195, 79)'
   }, {
     title: 'Header 4',
-    subTitle: 'Header rendered by  h4.',
+    subTitle: '#### Lorem Ipsum is simply ...',
     label: 'heading 4',
+    shortCut: `${COMMAND_KEY}4`,
     icon: header4Icon,
```

Comment:
The icons for H4-H6 are not updated and have a different style/color.

Reply:
H4-H6 are not finished, I'll replace it in the next Monday in another PR.